src/conn/conn_sweep.c

```diff
@@ -179,8 +240,28 @@ __sweep_server(void *arg)
 		WT_ERR(__wt_cond_wait(session, conn->sweep_cond,
 		    (uint64_t)conn->sweep_interval * WT_MILLION));
 
+		/*
+		 * Ignore in-use files once the open file count reaches the
+		 * minimum number of handles.
+		 */
+		if (conn->open_file_count < conn->sweep_handles_min)
+			continue;
```

Comment:
I'm concerned about this test: I can't convince myself that all of the paths through this code can't leave us with a handle that needs sweeping (and which is tying down a chunk of the cache), can't be left in place when the open-file-count drops, and we stop sweeping. I think we should not test this here, I think the test in `__sweep_mark_dead` is sufficient.